x-pack/plugin/transform/src/test/java/org/elasticsearch/xpack/transform/transforms/TransformFailureHandlerTests.java

```diff
@@ -63,9 +64,145 @@ public int getFailureCountChangedCounter() {
         }
     }
 
-    public void testUnattended() {
+    // TODO: Circuit breaking exception
```

Comment:
Can we remove this? Seems like we have coverage for circuit breaking exceptions now right?

Reply:
Good catch, I'll remove this line.